src/coreclr/src/debug/debug-pal/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ include_directories(../../pal/inc)
 
 add_definitions(-DPAL_STDCPP_COMPAT)
 
-if(WIN32)
+if(CLR_CMAKE_TARGET_WIN32)
```

Comment:
The following should be controlled by `HOST`.   It may collide with #28819

```
    add_definitions(-DWIN32_LEAN_AND_MEAN)
    include_directories(../../inc) #needed for warning control
```